static/app/utils/analytics/growthAnalyticsEvents.tsx

```diff
@@ -66,6 +66,7 @@ export type GrowthEventParameters = {
   'growth.sample_transaction_docs_link_clicked': {
     project_id: string;
   };
+  'growth.issue_open_in_discover_btn_clicked': {};
```

Comment:
I think we should have an event property if the feature is enabled or not so we can have this event in getsentry with different event properties

Reply:
the `issue_open_in_discover_btn_clicked` is sent only when the feature was enabled. If the feature was disabled we send `issue_open_in_discover_upsell_clicked` instead